WebRoot/js/ajax/dwt/xforms/XFormItem.js

```diff
@@ -994,14 +994,14 @@ XFormItem.prototype.handleKeyPressDelay = function(ev, domItem) {
 };
 
 XFormItem.prototype.getKeyPressHandlerHTML = function () {
-	// if AjxEnv.isNav, etc.
-	var keydownEv = DwtEvent.ONKEYPRESS;
+  // if AjxEnv.isNav, etc.
+  var keydownEv = DwtEvent.ONKEYPRESS; // "onkeypress"
```

Comment:
Previous changes wasn't made by me and I don't know how that was made.  See above comment/email exchange from Saket Patel.
I've made a new pull request including the comment '// "onkeypress"' just to make sure changes I need is in.